src/site/content/en/learn/css/css.11tydata.js

```diff
@@ -5,6 +5,8 @@ module.exports = function () {
     // e.g. A course with a key of 'a11y' would have a corresponding
     // _data/courses/a11y directory.
     projectKey: 'css',
+    tags: ['course-learn-css'],
+    searchTag: 'course-learn-css',
```

Comment:
We may want to put this in the _data/courses/css/meta.yml and then fetch them the same way we do the thumbnail down below in the eleventyComputed block. If the author has defined tags already on a course module you would likely need to merge them. Something like:

```
eleventyComputed: {
  tags: (data) => {
    const {projectKey} = data;
    return [...data.courses[projectKey].meta.tags, ...data.tags];
  }
}
```

I think it may also be good to see if we can consolidate these into a single property instead of two (tags and searchTag). Is that possible?

Reply:
I jus wonder- what is the rationale of having courses content defined in .yaml instead of .md and 11tydata, like for all the other types of content?

Reply:
I updated the tags to pull from meta.yml

Reply:
Mostly so we can consolidate into the _data directory. It's really helpful to have stuff at that top level when you need to, for instance, fetch data about a course from some other part of the site that isn't a direct descendant of the /learn/css/ folder.